starter-core/src/main/java/io/micronaut/starter/feature/testresources/EaseTestingFeature.java

```diff
@@ -36,7 +36,7 @@ protected EaseTestingFeature(TestContainers testContainers,
 
     @Override
     public void processSelectedFeatures(FeatureContext featureContext) {
-        if (!featureContext.isPresent(TestResources.class) && testResources != null) {
+        if (!featureContext.isPresent(TestContainers.class) && testResources != null) {
```

Comment:
Are you sure this is correct? I think the original intent was to check the presence of `TestResources` before adding it. It's an idiom repeated all over starter.

Reply:
The requirement is:

If they have selected `TestContainers` then don't apply `TestResources`

If they have selected nothing, apply `TestResources`

Previously we applied `TestResources` all the time, whether they had added `TestContainers` or not 🤔 

Do you have a link to the idiom?